voting/Cargo.toml

```diff
@@ -9,6 +9,7 @@ ccrypto = { package = "codechain-crypto", git = "https://github.com/CodeChain-io
 ckey = {package = "codechain-key", path = "../key"}
```

Comment:
The commit message seems not correct.
`Implement a service for the general_meeting module ` says that you implemented a service for the general meeting module. However, the service is used by other module and it is not used in the general meeting module.